src/components/inputs/JuiceTextInput.tsx

```diff
@@ -0,0 +1,14 @@
+import { Input, InputProps } from 'antd'
+import { classNames } from 'utils/classNames'
+
+export const JuiceInput = (props: InputProps) => {
```

Comment:
**suggestion (non-blocking** i don't mind `Juice` but also wanted to throw out `JB` as a shorter alternative, and also follows the contract name conventions (which may or may not be a good thing).

Reply:
I am fine with either. I have a bunch of PRs that add Juice related components so lets kick the can down the road